app/controllers/forms/page_controller.rb

```diff
@@ -67,7 +67,12 @@ def log_page_save(step, request, changing_existing_answer)
                     "page_save"
                   end
 
-      EventLogger.log_page_event(current_context, step, request, log_event)
+      EventLogger.log_page_event(current_context, step, request, log_event, skipped_question: skipped_question?(answers))
+    end
+
+    def skipped_question?(answers)
+      answers.each_value { |a| return false if a != "" }
```

Comment:
Think there are a couple of Rails syntax bits you might be able to use here:

```suggestion
      answers.any? { |a| a.present? }
```